privacy-policy.md

```diff
@@ -37,6 +39,7 @@ show_sidebar: false
 1.  <div style="text-align: justify">If you fill in open questions in the surveys/questionnaires in order to give us personal feedback, we might further contact you in order to get more information. This information may then be used with a purpose of improving the quality of OpenMR Benelux initiatives, or to invite you to events organized by OpenMR Benelux.</div>
 2.  <div style="text-align: justify">The summarized, anonymized statistics can also potentially be used to create open-access materials to be shared at our websites. This is a non-commercial activity, for informative purposes only, and aims at improving visibility of OpenMR Benelux activities.</div>
 3.  <div style="text-align: justify">If you deliver specific personal information such as dietary requirements while assigning for events, this can be used to better serve you at our events.</div>
+4.  <div style="text-align: justify">If you disclose personal information during a recorded meeting (e.g., via video or audio), we will try to make sure to delete those as much as possible before we make the recording publicly available. To prevent this from happening, we advise you to turn off your camera and microphone during such recordings. If you feel that we did not remove your personal details sufficiently, please reach out to us.</div>
```

Comment:
```suggestion
4.  <div style="text-align: justify">If you disclose personal information during a recorded meeting (e.g., via video or audio), we will try to make sure to delete those as much as possible before we make the recording publicly available. To prevent such personal details to be published online without your permission, we advise you to turn off your camera and microphone during such recordings. If you feel that we did not remove your personal details sufficiently, please reach out to us.</div>
```